src/main/java/Duke.java

```diff
@@ -6,5 +9,81 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("\t____________________________________________________________\n"
+                + "\tHello! I'm Duke\n" +
+                "\tWhat can I do for you?\n"
+                + "\t____________________________________________________________\n");
+
+
+        Task[] taskList = new Task[100];
+        int taskCount = 0;
+
+        Scanner in = new Scanner(System.in);
+        String line = in.nextLine();
+
+        while (!line.equals("bye")){
+            String[] words = line.split(" ",2); // split the first word from the rest of the sentence using space as the delimiter
```

Comment:
Line 25 of the code seems very long, maybe can consider placing the comment above line 25 to make the code tidier. But good job on explaining the the use of the function in your code. -SWJE